content/episodes/Scale and Scaling.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Scale vs Scaling"
+date: 2020-08-01T14:12:15+02:00
+lastmod: 2020-08-01T14:12:15+02:00
+episode: 12
+author: 'DevSecOps Talks'
+participants: ['Andrey', 'Julien']
+---
+
+Julian and Andrey got together to define the scale and ways to automate the scaling of your infrastructure in response to changes in load patterns.
```

Comment:
```suggestion
Julien and Andrey got together to define the scale and ways to automate the scaling of your infrastructure in response to changes in load patterns.
```

Reply:
@veggiemonk I told you it is going to happen :D